apps/developer-portal/src/components/user-stores/add-user-store.tsx

```diff
@@ -50,19 +61,33 @@ interface AddUserStoreProps {
  */
 export const AddUserStore = (props: AddUserStoreProps): ReactElement => {
```

Comment:
Missing type. Check other places as well.

```suggestion
export const AddUserStore: FunctionComponent<AddUserStoreProps> = (props: AddUserStoreProps): ReactElement => {
```